src/main/java/racinggame/domain/Judge.java

```diff
@@ -0,0 +1,29 @@
+package racinggame.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Judge {
+
+    private Judge() {}
+
+    public static List<String> getWinners(List<RacingCar> racingCars) {
+        int maximumPosition = getHighScore(racingCars);
+
+        return racingCars.stream()
+                .filter(car -> car.getPosition() == maximumPosition)
+                .map(RacingCar::getCarName)
+                .collect(Collectors.toList());
+    }
+
+    private static int getHighScore(List<RacingCar> racingCars) {
+        return Collections.max(getCarPositions(racingCars));
+    }
+
+    private static List<Integer> getCarPositions(List<RacingCar> racingCars) {
+        return racingCars.stream()
+                .map(car -> car.getPosition())
```

Comment:
```suggestion
                .map(RacingCar::getPosition)
```
메소드 참조 방식으로도 표현가능합니다.

Reply:
감사합니다! 반영했습니다!